crawler/dataParser.py

```diff
@@ -80,6 +90,50 @@ def runParser(self):
     for i in range(0, self.MAX_THREADS):
       threadPool[i].join()
 
+    FileIO.writeJsonFile(self.outlinkGraph.nodes, self.outlinkGraphFile)
+    FileIO.writeJsonFile(self.inlinkGraph.nodes, self.inlinkGraphFile)
+
+  def findNewLinks(self, parseLink):
```

Comment:
I think function is redundant. All the anchor tags are already being extracted via extract data, its probably faster the set of anchors as a second object, then process that output in dataparser